tests/test_configOverrides.py

```diff
@@ -255,9 +239,23 @@ def testChoiceValueInt(self):
         with self.assertRaises(pexConfig.FieldValidationError):
             self.checkSingleFieldOverride(field, [0, 1])
 
-        # non-parseable input
-        with self.assertRaises(RuntimeError):
-            self.checkSingleFieldOverride(field, "value")
+    def testSettingVar(self):
+        """Test setting a field with a string that represents a variable name
+        """
+        field = "fChoiceInt"
+
+        # verify loading variable
+        test_choice_value = 1  # noqa: F841
+        self.checkSingleFieldOverride(field, "test_choice_value", 1)
+
+        # Verify That python importing a variable works
+        config = ConfigTest()
+        overrides = ConfigOverrides()
+        overrides.addPythonOverride("from numpy import pi")
+        overrides.addValueOverride("fFloat", "pi")
+        overrides.applyTo(config)
+        from numpy import pi
```

Comment:
I know `numpy` is practically required dependency now but maybe test can be done with `math.pi` instead to avoid this import?

Reply:
sure, that would help import times too I guess.